pkgs/applications/editors/emacs/generic.nix

```diff
@@ -74,7 +72,7 @@
 , WebKit
 
 # Boolean flags
-, nativeComp ? true
+, withNativeCompilation ? true
```

Comment:
This might break users.

Reply:
A "trace message" can help.

Reply:
I am inclined to let it break.
Indeed I would like a "graceful break", something warning the user about this new option.